src/common/history.h

```diff
@@ -25,6 +25,14 @@
 struct dt_develop_t;
 struct dt_iop_module_t;
 
+typedef enum dt_hash_history_t
+{
+  DT_HH_INITIAL = 1 << 0,
+  DT_HH_DEFAULT = 1 << 1,
+  DT_HH_CURRENT = 1 << 2,
+  DT_HH_UNKNOWN = 1 << 3
+} dt_hash_history_t;
```

Comment:
type should be : dt_history_hash_t (see below)

Reply:
There is a dt_history_xxx, dt_dev_history_xxx, it's why I've added a dy_hash_history_xxx.

Reply:
It's look for me more consistent as is, but I can change. Let me know.

Reply:
done